app/src/main/java/org/wikipedia/search/SearchResult.kt

```diff
@@ -0,0 +1,46 @@
+package org.wikipedia.search
+
+import org.wikipedia.dataclient.WikiSite
+import org.wikipedia.dataclient.mwapi.MwQueryPage
+import org.wikipedia.model.BaseModel
+import org.wikipedia.page.PageTitle
+
+class SearchResult : BaseModel {
```

Comment:
I see you removed the inheritance from `Parcelable`, which I suppose is good, but can we also remove the other weird bits like `hashCode()`, `equals()`, and `toString()`?